### PR TITLE
Website docs for Vault EKM provider for MS SQL

### PR DESCRIPTION
**Pull request by tomhjp**
Provides overview, installation and troubleshooting docs for the upcoming Vault EKM provider for MS SQL server

### CODE REVIEW

File: website/content/docs/platform/mssql/installation.mdx [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+1. Download and run the latest Vault EKM provider installer from
+  [releases.hashicorp.com](https://releases.hashicorp.com/vault-mssql-ekm-provider/)
+
+1. Enter your Vault server's address when prompted and complete the installer
+
+### Configure the EKM provider using SQL
+
+1. Enable the EKM feature and create a cryptographic provider using the folder
+   you just installed the EKM provider into.
+
```
Comment (tvoran) [inline website/content/docs/platform/mssql/installation.mdx:96]: Is there any more info here we should provide for where these commands are run from? Is this just in a SQL session or something?

File: website/content/docs/platform/mssql/installation.mdx [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    RECONFIGURE;
+    GO
+
+    CREATE CRYPTOGRAPHIC PROVIDER TransitVaultProvider
+    FROM FILE = 'C:\Program Files\HashiCorp\Transit Vault EKM Provider\TransitVaultEKM.dll'
+    GO
+    ```
+
+1. Next, create credentials for an admin to use EKM with your AppRole role and
```
Comment (tvoran) [inline website/content/docs/platform/mssql/installation.mdx:120]: Does the `with IDENTITY` parameter correspond to the AppRole role id? I wonder if there's an easy way to make it a little more clear what the parameters in the sql snippet correlate to in vault, like with comments or something.

File: website/content/docs/platform/mssql/installation.mdx [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+    USE TestTDE;
+    GO
+
+    CREATE DATABASE ENCRYPTION KEY
+    WITH ALGORITHM = AES_256
+    ENCRYPTION BY SERVER ASYMMETRIC KEY TransitVaultAsymmetric;
+    GO
+
+    ALTER DATABASE TestTDE
+    SET ENCRYPTION ON ;
```
Comment (tvoran) [inline website/content/docs/platform/mssql/installation.mdx:178]: Is the space before the semicolon intentional?

File: website/content/docs/platform/mssql/troubleshooting.mdx [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+true, for example:
+
+```json
+{
+    "vaultApiBaseUrl":  "http://vault.example.com:8200",
+    "enableTrace": true
+}
+```
+
+Restart the server for the config change to take effect, and you should see more
```
Comment (tvoran) [inline website/content/docs/platform/mssql/troubleshooting.mdx:29]: Which server? The database server? Or is it just the provider?

**Review by tvoran — APPROVED** [review]
Love that there's a troubleshooting section! Just some small, non-blocking questions.

File: website/content/docs/platform/mssql/installation.mdx (hunk 1 above)
Comment (tomhjp) [reply website/content/docs/platform/mssql/installation.mdx:108]: Good point, added a little intro paragraph for this section.

File: website/content/docs/platform/mssql/installation.mdx (hunk 2 above)
Comment (tomhjp) [reply website/content/docs/platform/mssql/installation.mdx:132]: Yeah good point, I've made it a bit clearer now.

File: website/content/docs/platform/mssql/installation.mdx (hunk 3 above)
Comment (tomhjp) [reply website/content/docs/platform/mssql/installation.mdx]: Nope! Good spot :)

File: website/content/docs/platform/mssql/troubleshooting.mdx (hunk 4 above)
Comment (tomhjp) [reply website/content/docs/platform/mssql/troubleshooting.mdx]: SQL Server sadly - I've not found a less intrusive and reliable way to fully reload the dll yet. Clarified in the doc

**Pull request merged**